src/utils.js

```diff
@@ -50,7 +50,8 @@ async function setUpNpmConfig (registry, strictSSL) {
     rollback: false,
     fund: false,
     'strict-ssl': strictSSL,
-    noproxy: 'registry.npmjs.org'
+    noproxy: 'registry.npmjs.org',
+    cafile: process.env.CA_FILE
```

Comment:
@tianfeng92 are there any side effect if this env var is not defined?

Reply:
tested in docker mode. no side effect when signing `undefined` to cafile. Let me add more check before this line

Reply:
no.. actually it doesn't work. investigating

Reply:
works.